ci/travis-install.sh

```diff
@@ -110,7 +110,7 @@ case "$ci_distro" in
             libcurl4-openssl-dev \
             procps \
             zlib1g-dev \
-            python-yaml \
+            python{,3}-yaml \
```

Comment:
python3-yaml only should be enough on Debian/Ubuntu now that we're using python3 everywhere. python-yaml (which means Python 2) isn't in the Build-Depends any more for the production builds in Debian, and those work fine.